File: Cau3_AndroidAppIntroduceMyseft/app/src/main/java/ntu/a63131916/MainActivity.java
package ntu.a63131916;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private ImageView logoImageView;
    private TextView introduceTextView;
    private Button watchTrailerButton;
    private TextView projectDateTextView;
    private ImageView facebookIconImageView, instagramIconImageView, twitterIconImageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Initialize views
        logoImageView = findViewById(R.id.logoImageView);
        introduceTextView = findViewById(R.id.introduceTextView);
        watchTrailerButton = findViewById(R.id.watchTrailerButton);
        projectDateTextView = findViewById(R.id.projectDateTextView);
        facebookIconImageView = findViewById(R.id.facebookIconImageView);
        instagramIconImageView = findViewById(R.id.instagramIconImageView);
        twitterIconImageView = findViewById(R.id.twitterIconImageView);
    }
}

File: ArrayList/app/src/main/java/vn/edu/arraylist/MainActivity.kt
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import java.util.Arrays;
import java.util.List;

public class MainActivity extends AppCompatActivity {

    String[] arrstring = new String [] {"Mai","Cuc","Lan","Hong"};
    Integer[] arrInteger = new Integer[] {1,2,1,34,5,78,234};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        List<String> list = Arrays.asList(arrString);
        List<Integer> integerList = Arrays.asList(arrInteger);
    }

}